distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json.enso

```diff
@@ -31,6 +30,7 @@ from project.Data.Text.Extensions import all
 from project.Metadata.Choice import Option
 from project.Metadata.Widget import Single_Choice
 
+polyglot java import com.fasterxml.jackson.core.JsonLocation
```

Comment:
@hubertp, because _native image_ isn't really a friend of reflective access, I tried to eliminate as much of usage of `asGuestValue` as possible. Right now we are only using it for wrapping exception (and leaking EnsoContext):

```
enso/engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoFile.java
  687,23:     var guestEx = ctx.asGuestValue(ex);

enso/engine/runtime/src/main/java/org/enso/interpreter/runtime/scope/TopLevelScope.java
  169,22:       return context.asGuestValue(context);

enso/engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java
  492,17:   public Object asGuestValue(Object obj) {
  493,24:     return environment.asGuestValue(obj);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/MinusBuiltinDateTimeNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/MinusBuiltinDurationNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/MinusBuiltinTimeOfDayNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/NewBuiltinDateNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/NewBuiltinDateTimeNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/NewBuiltinTimeOfDayNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/NewBuiltinTimeZoneNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/ParseBuiltinTimeZoneNode.java
  35,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/PlusBuiltinDateTimeNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/PlusBuiltinDurationNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/date/PlusBuiltinTimeOfDayNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/immutable/NewArrayProxyBuiltinArrayLikeHelpersNode.java
  34,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/CopyBuiltinFileNode.java
  36,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/CreateDirectoryBuiltinFileNode.java
  28,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/CreationTimeBuiltinFileNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/DeleteBuiltinFileNode.java
  28,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/InputStreamBuiltinFileNode.java
  35,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/LastModifiedTimeBuiltinFileNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/ListImmediateChildrenArrayFileNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/MoveBuiltinFileNode.java
  36,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/OutputStreamBuiltinFileNode.java
  35,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/PosixPermissionsBuiltinFileNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/ReadLastBytesBuiltinFileNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/io/SizeBuiltinFileNode.java
  27,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/engine/runtime/target/scala-2.13/src_managed/main/org/enso/interpreter/node/expression/builtin/CreateProcessSystemNode.java
  35,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);
  38,54:       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);

enso/lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/ExecuteMethodImplGenerator.java
  145,23:               "      .asGuestValue(" + qual + "." + name + "(" + paramsApplied + "));"

enso/lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/NoSpecializationClassGenerator.java
  38,9:     var asGuestValue = needsGuestValueConversion(origin);
  39,9:     if (asGuestValue
  47,40:         processingEnvironment, origin, asGuestValue, varArgExpansion);

enso/lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/SafeWrapException.java
  36,63:           "    throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);");

enso/lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/SpecializedMethodsGenerator.java
  293,60:             methodBody.add("  return EnsoContext.get(this).asGuestValue(result);");

enso/lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/Builtin.java
  256,59:    *       throw new PanicException(EnsoContext.get(this).asGuestValue(e), this);"
```
the usage of `asGuestValue` for exceptions is still needed as 1bae3745c6544b60eef4dcca9c8bd7a2065be517 shows.